package.json

```diff
@@ -5,6 +5,11 @@
   "main": "index.js",
   "scripts": {
     "dev": "pm2 start process-dev.json",
+    "local": "nodemon ./dist/src/app.js",
```

Comment:
do we need need to use nodemon? can we use ts-node https://www.npmjs.com/package/ts-node ?

Reply:
Removed